toolkit/templates/forms/_checkbox-tree-selection-buttons.html

```diff
@@ -1,30 +1,30 @@
-<fieldset class="checkbox-tree__selection-buttons">
+<fieldset class="checkbox-tree__inputs" id="checkbox-tree__inputs">
```

Comment:
The problem is that the javascript that runs alongside this has been making the assumption that there's only one category-picker on a page, and, for G9, that's all there will be. 

The example page in the toolkit has a bunch of them on a page, so that's where this becomes a problem.  

Right now we need the javascript working in the supplier app against the real-live category-picker more than we need the javascript to work in the frontend toolkit example page.